firefox-ios/firefox-ios-tests/Tests/XCUITests/BrowsingPDFTests.swift

```diff
@@ -38,7 +38,7 @@ class BrowsingPDFTests: BaseTestCase {
         waitUntilPageLoad()
 
         // Click on a link on the pdf and check that the website is shown
-        app.links.element(boundBy: 0).tapOnApp()
+        app.webViews.links.element(boundBy: 0).tapOnApp()
```

Comment:
There are also `links` covered by the PDF reader. The test should mean the link on the PDF doc.